docs/getting-started/quickstart.md

```
@@ -1,55 +1,139 @@
-- Check out our [FAQ](/faq)
-- Contact [support@coderabbit.ai](mailto:support@coderabbit.ai)
+Use your usual Git workflow to perform the following steps in the `coderabbit-test` repository:
 
-:::tip
-CodeRabbit starts working immediately. You can customize its behavior later when needed.
-:::
+1. Create a branch named `add-utils`.
+
+1. In that new `add-utils` branch, create a new file called `simple_utils.py`, with the following content:
+
+	```
```

markdownlint-cli2 (0.17.2)

Hard tabs
Column: 1

(MD010, no-hard-tabs)